packages/react-hooks/README.md

```diff
@@ -187,6 +187,47 @@ const Example = React.forwardRef(function Example(props: {}, forwardedRef: React
 });
 ```
 
+## useMount
+
+```ts
+const useMount: (callback: () => void) => void;
+```
+
+Hook which asynchronously executes a callback once the component has been mounted.
```

Comment:
Might be good to note in the readme which hook is used under the hood (and link to the docs for that hook).